embedded-ebpf.go

```diff
@@ -1,7 +1,7 @@
 //go:build ebpf
 // +build ebpf
 
-package tracee
+package ebpf
```

Comment:
can you please remind me why we have to keep the embedded* files in the root dir and not move it under pkg/ebpf?

Reply:
Go embed package does not support relative paths like './../somebpf.o'

Reply:
Yes, I just saw the issue saying exactly that.
But it seems that there is a workaround for this (which is not portable to windows, but that is not a problem for us): https://github.com/golang/go/issues/46056#issuecomment-938251415

Reply:
Do we need yet another workaround? :)

Reply:
TBH I don't know...
It just looks ugly to have this go file in the root directory

Reply:
> because of ./dist/btfhub path. it needs to be in root because embed does not accept "../../" path.

Yes, @danielpacak already mentioned this - see my reply above